frontend/src/modules/vehicles/VehicleLocation.vue

```diff
@@ -4,14 +4,106 @@
       <CCardBody>
         <h4>Location</h4>
         <hr />
+        <GmapMap
+          :center="center"
+          :zoom="11"
+          style="height: 400px"
+        >
+          <GmapInfoWindow :options="infoOptions" :position="infoWindowPos" :opened="infoWinOpen" @closeclick="infoWinOpen=false">
+            <CLink :href="infoLink" target="_blank">{{infoContent}}</CLink>
```

Comment:
In line 13 in VehicleLocation, there is a security hotspot as we are vulnerable to XSS attacks due to bypassing the auto-escaping Vue features by not having the href inside a child node.

Reply:
Line 12 - 14 have been removed since they don't add anything to the placeholder code for now.